teuthology/task/install.py

```diff
@@ -776,10 +740,9 @@ def install(ctx, config):
 
     project = config.get('project', 'ceph')
 
-    global deb_packages
-    global rpm_packages
-    debs = deb_packages.get(project, [])
-    rpm = rpm_packages.get(project, [])
+    global PACKAGES
```

Comment:
I can appreciate that concern.  At some point we should start removing cruft like this though.

FWIW, I see PACKAGES being used in install.upgrade_common, but it doesn't seem to depend on  any mutation that happens in install.install caused by it being global.